app/models/amendments/articulated_amendment.rb

```diff
@@ -12,9 +12,5 @@ def articulated?
     def allow_articulated?
       true
     end
-
-    def section
```

Comment:
Why does the section is not needed anymore?

Reply:
Beacuse of this https://github.com/podemos-info/national_budgets/blob/61be6fee814689f8b240546996786f641d6f0dc9/app/models/amendments/articulated_amendment.rb#L6